deploy/crds/shipwright.io_buildruns.yaml

```diff
@@ -64,19 +68,128 @@ spec:
                 required:
                 - name
                 type: object
+              env:
+                description: Env contains additional environment variables that should
+                  be passed to the build container
+                items:
+                  description: EnvVar represents an environment variable present in
+                    a Container.
+                  properties:
+                    name:
+                      description: Name of the environment variable. Must be a C_IDENTIFIER.
+                      type: string
+                    value:
+                      description: 'Variable references $(VAR_NAME) are expanded using
```

Comment:
@coreydaley did you generate CRDs directly from your fedora workstation, or did you employ an ubuntu based container ?

this looks like you've hit the line break issue https://github.com/shipwright-io/build/issues/832

At least for me, I was able to craft an image docker.io/gmontero/ubuntu-with-go-controller-gen, launched a container where I mounted my local branch, and run make generate-crds from it to craft these files with the desired line  breaking

Reply:
Yes, I did.

Reply:
yeah assuming that meant you did do it on fedora then ok then you'll need to redo this from ubuntu like I mentioned ^^

I think I also heard @HeavyWombat say that he got it to work right on RHEL (though I don't remember if he indicated when version of RHEL he used)

Reply:
CRDs should all be correctly formatted now

Reply:
yep look good now